control_system_stack/setyaw/src/test_client.py

```diff
@@ -0,0 +1,59 @@
+#! /usr/bin/env python
+
+PKG = 'setyaw'
+import roslib; roslib.load_manifest(PKG)
+import rospy
+
+# Brings in the SimpleActionClient
+import actionlib
+
+# Brings in the messages used by the fibonacci action, including the
+# goal message and the result message.
+from kraken_msgs.msg import setYawAction as actionMessageSetYawAction
+from kraken_msgs.msg import setYawFeedback
+from kraken_msgs.msg import setYawResult
+from kraken_msgs.msg import setYawGoal
+
+def serverSentSomeFeedback(feedback_obj):
+
+    rospy.loginfo('Recieved feedback from the server, Desired_yaw : %f', feedback_obj.Desired_yaw)
```

Comment:
@aniket11bh I don't think that the all these values are required at every iteration. Just the error should do. The error is basically the difference between Desired and Current yaw anyway.